src/exe/colmap.cc

```diff
@@ -1633,23 +1633,37 @@ int RunPointTriangulator(int argc, char** argv) {
 //       {
 //           "camera_id": 1,
 //           "image_prefix": "left1_image"
+//           "location": [0, 0, 0],
+//           "orientation": [1, 0, 0, 0]
 //       },
 //       {
 //           "camera_id": 2,
 //           "image_prefix": "left2_image"
+//           "location": [0, 0, 0],
+//           "orientation": [0, 1, 0, 0]
 //       },
 //       {
 //           "camera_id": 3,
 //           "image_prefix": "right1_image"
+//           "location": [0, 0, 0],
+//           "orientation": [0, 0, 1, 0]
 //       },
 //       {
 //           "camera_id": 4,
 //           "image_prefix": "right2_image"
+//           "location": [0, 0, 0],
+//           "orientation": [0, 0, 0, 1]
 //       }
 //     ]
 //   }
 // ]
 //
+// The "camera_id" and "image_prefix" fields are required, whereas the
```

Comment:
* It would be good to call location->rel_tvec and orientation->rel_qvec for consistency with the rest of the code base.
* It would be good to describe the coordinate convention of these parameters, i.e., rel_qvec/rel_tvec transform coordinates from rig to camera coordinate space.

Reply:
Renamed the fields and added a bit more to the comment below. Let me know if more explanation is needed.